src/tilesource.js

```diff
@@ -219,13 +219,14 @@ $.TileSource = function( width, height, tileSize, tileOverlap, minLevel, maxLeve
 
         this.tileOverlap = options.tileOverlap ? options.tileOverlap : 0;
         this.minLevel    = options.minLevel ? options.minLevel : 0;
-        this.maxLevel    = ( undefined !== options.maxLevel && null !== options.maxLevel ) ?
+        this.setMaxLevel( ( undefined !== options.maxLevel && null !== options.maxLevel ) ?
```

Comment:
We should leave this as `this.maxLevel =` as it should work that way (since lots of other code paths in other tile sources and even in this file do so).